### PR TITLE
Rewrite contribution guide

### PR DESCRIPTION
**Pull request by Loctav**
Please fix my horrendous English.

### CODE REVIEW

File: wiki/osu!wiki_contribution_guide/en.md [hunk 1]
```diff
@@ -1,936 +1,80 @@
...
-[Matt Liu image source]: http://www.slideshare.net/MatthewKLiu/git-branch-management "Git branch management by Matt Liu"
-[Git set default editor]: https://swcarpentry.github.io/git-novice/02-setup/ "swcarpentry on setting up the default editor"
-[Githowto resolve conflicts]: https://githowto.com/resolving_conflicts "Githowto's 30. Resolving Conflicts"
-[git mergetool help by karenyyng]: https://gist.github.com/karenyyng/f19ff75c60f18b4b8149 "karenyyng's git mergetool guide"
-[Ruslan Osipov's vimdiff tutorial]: http://www.rosipov.com/blog/use-vimdiff-as-git-mergetool/ "Ruslan Osipov's vimdiff tutorial"
-[GH Blog GPG]: https://github.com/blog/2144-gpg-signature-verification "GitHub blog on GPG signature verification"
-[GPG guide by Duncan]: http://mrexodia.cf/guide/2016/05/30/Github-Gpg "mrexodia blog post on GitHub GPG by Duncan"
+This quick-start guide is intended for newcomers that never worked with GitHub and the wiki contribution workflow before. If you have previous experience using GitHub, feel free to skip this guide and follow the common feature-branch workflow for content updates in this repository.
```
Comment (TPGPL) [inline wiki/osu!wiki_contribution_guide/en.md]: for newcomers that have never worked

Comment (TPGPL) [inline wiki/osu!wiki_contribution_guide/en.md]: previous experience?

File: wiki/osu!wiki_contribution_guide/en.md [hunk 2]
```diff
@@ -1,936 +1,80 @@
...
+    - Go to the [osu-wiki repository page](https://github.com/ppy/osu-wiki) and click "Fork" in the upper-right corner of the page and wait until it is finished. You will then be led to the page of your forked repository.
 
-# How to contribute to the osu!wiki
+      ![](img/fork-repository.png "Click the Fork button to make your own copy")
 
-Thanks for your interest in making the osu!wiki better!
-As you can see, everything has been moved to [GitHub][GitHub osu-wiki].
+3. [Download and install GitHub Desktop](https://desktop.github.com) or another tool to use Git. This guide leads you through the workflow with GitHub Desktop only though.
+4. Run GitHub Desktop and sign in to GitHub.com after installation.
+5. Clone your forked repository to your local PC. For that, go to the page of your forked repository and click the green "Clone or download" button. Copy the shown URL from it and use it
```
Comment (TPGPL) [inline wiki/osu!wiki_contribution_guide/en.md]: `Clone or download` button

File: wiki/osu!wiki_contribution_guide/en.md [hunk 3]
```diff
@@ -1,936 +1,80 @@
...
-To contribute to the osu!wiki, you must have the following:
+  ![](img/command-prompt-1.png "Enter the mentioned commands into the command prompt and press ENTER")
 
-- A [GitHub account][GH account].
-- Knowledge in [Markdown][MD], specifically [GitHub-Flavoured Markdown(GFM)][GFM].
-- [Understanding the Git's workflow (feature-branch workflow)][GH Help feature branch].
-  - [A simplified visual explanation provided by GitHub Guides][GH Guide feature branch].
-- Able to write in English, preferably British English for consistency.
-  - Everything not related to the page content itself (commit messages, pull requests, filenames, etc) **must** be written in English.
+4. To update your forked repository, enter `git pull upstream master` into the command prompt.
```
Comment (TPGPL) [inline wiki/osu!wiki_contribution_guide/en.md]: not `git merge upstream/master`?

File: wiki/osu!wiki_contribution_guide/en.md [hunk 4]
```diff
@@ -1,936 +1,80 @@
...
-  - Some editors (like Visual Studio Code) may provide a Git interface.
+Now you can **start working on the osu!wiki files**. For that, go to the folder you specified when cloning the forked repository. There you will find all articles and files that are used in the osu!wiki. Use any editor of your choice to make the changes you desire. Keep in mind that the osu!wiki uses [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/) for formatting purposes. Make sure to follow the [Article Style Guide](/wiki/Article_Style_Guide) when making changes.
 
-## Forking osu-wiki repository
+In order to distribute your changes, do the following:
 
-To make changes to the osu-wiki repository, you must first create your own copy of the repository.
-This is known as "forking".
+1. Save the files you made changes at.
+2. Go to the GitHub Desktop client. You will notice that your change will be visible.
```
Comment (TPGPL) [inline wiki/osu!wiki_contribution_guide/en.md]: will notice that your changes are visible

File: wiki/osu!wiki_contribution_guide/en.md [hunk 5]
```diff
@@ -1,936 +1,80 @@
...
+  ![](img/push-commits.png "Pressing this button will upload all changes to GitHub")
 
-Now click at your profile and GitHub would redirect you to where the fork is.
+## How to request your changes to be amended
 
-Finally, you should see something like this.
+In order to get your changes into the official osu!wiki itself, it is required to create *Pull Requests* from your forked repository into the original one. This is nothing more but a request to check your changes and take them over into the official version of the wiki. Your changes will then be reviewed and eventually amended or rejected.
 
-![Fork Snapshot](./img/Contribute_ownrepo.jpg "Fork Snapshot")
+1. Go to the page of your forked repository and click "New Pull Request".
```
Comment (TPGPL) [inline wiki/osu!wiki_contribution_guide/en.md]: code the buttons instead of quoting?

File: wiki/osu!wiki_contribution_guide/en.md [hunk 6]
```diff
@@ -1,936 +1,80 @@
...
 
-Congratulations!
-You have made a copy of the osu-wiki repository!
+  ![](img/pr-1.png)
 
-You can now proceed to make changes.
+2. Click the green "Create pull request" button in the yellow box.
```
Comment (TPGPL) [inline wiki/osu!wiki_contribution_guide/en.md]: button

File: wiki/osu!wiki_contribution_guide/en.md [hunk 7]
```diff
@@ -1,936 +1,80 @@
...
 
-## Making Changes
+  ![](img/pr-2.png)
 
-### Concept of feature-branch workflow
+3. Give your *Pull Request* a proper title and a description. Then click "Create pull request". This will open the Pull Request and lead you to the page of it. Other contributors can now comment and review these changes.
```
Comment (TPGPL) [inline wiki/osu!wiki_contribution_guide/en.md]: button

File: wiki/osu!wiki_contribution_guide/en.md (hunk 3 above)
Comment (Loctav) [inline wiki/osu!wiki_contribution_guide/en.md]: I followed some guide here. I am not super save with direct git commands, so I have to trust someone whoever knows better than me

File: wiki/osu!wiki_contribution_guide/en.md [hunk 8]
```diff
@@ -1,936 +1,80 @@
...
-[Matt Liu image source]: http://www.slideshare.net/MatthewKLiu/git-branch-management "Git branch management by Matt Liu"
-[Git set default editor]: https://swcarpentry.github.io/git-novice/02-setup/ "swcarpentry on setting up the default editor"
-[Githowto resolve conflicts]: https://githowto.com/resolving_conflicts "Githowto's 30. Resolving Conflicts"
-[git mergetool help by karenyyng]: https://gist.github.com/karenyyng/f19ff75c60f18b4b8149 "karenyyng's git mergetool guide"
-[Ruslan Osipov's vimdiff tutorial]: http://www.rosipov.com/blog/use-vimdiff-as-git-mergetool/ "Ruslan Osipov's vimdiff tutorial"
-[GH Blog GPG]: https://github.com/blog/2144-gpg-signature-verification "GitHub blog on GPG signature verification"
-[GPG guide by Duncan]: http://mrexodia.cf/guide/2016/05/30/Github-Gpg "mrexodia blog post on GitHub GPG by Duncan"
+This quick-start guide is intended for newcomers that have never worked with GitHub and the wiki contribution workflow before. If you already have experience using GitHub, feel free to skip this guide and follow the common feature-branch workflow for content updates in this repository.
```
Comment (TPGPL) [inline wiki/osu!wiki_contribution_guide/en.md]: not sure about `have experience using GitHub`
I would change that `using` into `with` or `in`; probably wait for Eph

File: wiki/osu!wiki_contribution_guide/en.md (hunk 3 above)
Comment (LiquidPL) [inline wiki/osu!wiki_contribution_guide/en.md]: `git pull upstream master` appears to be a shorthand of:
```
git fetch upstream/master
git merge upstream/master
```

Personally I'd remove the mention of `git fetch upstream` from step 3 and just tell people to fetch a specific branch whenever they want to merge it into a repo.

Comment (kj415j45) [inline wiki/osu!wiki_contribution_guide/en.md]: Add:
>Make sure you are on `` `master` `` branch

it's very important

File: wiki/osu!wiki_contribution_guide/en.md [hunk 9]
```diff
@@ -1,936 +1,100 @@
...
-<!-- Programs and Extensions -->
-[VSC]: https://code.visualstudio.com "Visual Studio Code"
-[Brackets]: http://brackets.io/ "Brackets"
-[Atom]: https://atom.io "Atom"
-[Shiba]: https://github.com/rhysd/Shiba "Shiba"
-[Git]: https://git-scm.com/ "Git"
-[Git Extensions]: https://gitextensions.github.io/ "Git Extensions"
+1. [Sign up a GitHub account](https://github.com/join?source=header-home) if you do not have one already.
+2. Fork the osu-wiki repository. This creates your very own copy of the osu!wiki files that you can edit freely without worries.
+    - Go to the [osu-wiki repository page](https://github.com/ppy/osu-wiki) and click `Fork` in the upper-right corner of the page and wait until it is finished. You will then be led to the page of your forked repository.
```
Comment (peppy) [inline wiki/osu!wiki_contribution_guide/en.md]: don't use second level bullet points. just put it in its own paragraph.

File: wiki/osu!wiki_contribution_guide/en.md [hunk 10]
```diff
@@ -1,936 +1,100 @@
...
 
-# How to contribute to the osu!wiki
+      ![](img/fork-repository.png "Click the Fork button to make your own copy")
 
-Thanks for your interest in making the osu!wiki better!
-As you can see, everything has been moved to [GitHub][GitHub osu-wiki].
+3. [Download and install GitHub Desktop](https://desktop.github.com) or another tool to use Git. This guide leads you through the workflow with GitHub Desktop only though.
+4. Run GitHub Desktop and sign in to GitHub.com after installation.
+5. Clone your forked repository to your local PC. For that, go to the page of your forked repository and click the green `Clone or download` button. Copy the shown URL from it and use it
```
Comment (peppy) [inline wiki/osu!wiki_contribution_guide/en.md]: click the "open in desktop" button rather than copy + paste

**Comment by peppy** [comment]
Split the guide into two parts maybe. The first part should guide through making changes via the github website. This method should be recommended for anyone just wanting to work on a single page, as it makes things much simpler and provides all the features / forking necessary to get started with zero overhead.

File: wiki/osu!wiki_contribution_guide/en.md (hunk 8 above)
Comment (TPGPL) [inline wiki/osu!wiki_contribution_guide/en.md]: Have never worked with wiki contribution?

**Comment by sr229** [comment]
Don't forget to add to link the [Guide for Pull Request Reviews](https://help.github.com/articles/approving-a-pull-request-with-required-reviews/) for new users!

**Comment by sr229** [comment]
I might "hijack" this PR in a later time so this would be mergable

**Comment by Ephemeralis** [comment]
Inactive PR, closing - feel free to reopen later on.

**Pull request closed without merging**